lib/renewals/schema.ts

```diff
@@ -0,0 +1,8 @@
+export default `
```

Comment:
Can we add a TODO to the top of this file saying that this schema is not finalized and there will be changes following the creation of the `createRenewalApplication` API?